todo_app/templates/index.html

```diff
@@ -45,7 +46,7 @@ <h2>Items</h2>
         const newItem = document.getElementById('new_item')
         const newNode = document.createElement("li")
         newNode.id = "li" + item.id
-        newNode.innerHTML = "<span>" + item.title + " - </span><span id=\"status" + item.id +")\" onclick=\"updateStatus("+ item.id +", event)\">"+ item.status + "</span> <span onclick=\"deleteItem(" + item.id + ")\">(Delete)</span>"
+        newNode.innerHTML = "<span>" + item.title + " - </span><span id=\"status" + item.id +"\" onclick=\"updateStatus('"+ item.id +"', event)\">"+ item.status + "</span> <span onclick=\"deleteItem('" + item.id + "')\">(Delete)</span>"
```

Comment:
Yeah totally agree, I added the REST calls to avoid reloading the page after adding a create_item feature. I was migrating towards SPA+API but was concerned I'd veer too far from the project exercise and so didn't change the initial item creation in this PR (I did start towards it as part of the stretch goals but ran out of time). I'll look to tidy this up as part of module 3 exercise.